node/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+  "compilerOptions": {
+    "types": ["node"],
+    "target": "es2019",
+    "module": "commonjs",
+    "sourceMap": false,
+    "allowJs": true,
+    "noImplicitThis": true,
+    "moduleResolution": "node",
+    "skipLibCheck": true,
+    "alwaysStrict": true,
+    "esModuleInterop": true,
+    "noImplicitAny": true,
+    "noImplicitReturns": true,
+    "strictFunctionTypes": true,
+    "strictNullChecks": true,
+    "strictPropertyInitialization": true
+  },
```

Comment:
Have you considered extending the VTEX tsconfig?

Reply:
I haven't, but I can do this on a follow-up PR!